src/rsem_quant.py

```diff
@@ -60,6 +62,20 @@ def format_endedness(endedness):
         return ''
 
 
+def calculate_number_of_genes_detected(quant_tsv):
+    """Calculates number of rows where value on the column TPM is greater than 1.
+
+    Args:
+        quant_tsv: filename of a .tsv of RSEM quants
+
+    Returns:
+        int number_of_genes_detected: number of entries > 1 in TPM column
+    """
+    quants = pd.read_csv(quant_tsv, sep='\t')
+    number_of_genes_detected = sum(quants['TPM'] > 1)
```

Comment:
Do you think it would be easier for a reader if the cutoff was surfaced as a parameter?  The way you do it here has the advantage of locking down the way the metric is calculated, which is good for standardization.

Reply:
I think actually the best way would be to have the threshold as an argument for the function with default being 1. I would not give an option to set it from the workflow level though. This would be more readable, and still preserve the 'hardcoded by working group' nature of the value.